src/CSharp/EasyMicroservice.MemoryPack/EasyMicroservice.MemoryPack.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Please remove ImplicitUsings line.

Reply:
thank you for attention  . 
conversion solved